src/index.js

```diff
@@ -1,23 +1,84 @@
+/* @flow */
 /* eslint prefer-template: 0 */
 import React from 'react';
 import accepts from 'attr-accept';
 import getDataTransferItems from './getDataTransferItems';
 
+type InputAttributes = {
+  accept: string,
+  type: string,
+  style: Object,
+  multiple: boolean,
+  name?: string,
+  ref: Function,
+  onChange: Function
+};
+
+export type Props = {
+  onClick: Function,
```

Comment:
what kind of Function is this? Make this explicit!
E.g. `onClick: (elementId: string) => void`